### ISSUE TITLE
verifying mitmproxy works?

### ISSUE THREAD

**Issue by alias6466** [cuckoosandbox/cuckoo#1801]
Hey all,

I've installed MITMPROXY with the following commands

sudo apt-get install python3-pip python3-dev libtiff5-dev libjpeg8-dev libwebp-dev
sudo pip3 install mitmproxy

mitmproxy
cd ~/.mitmproxy
cp mitmproxy-ca-cert.p12 /home/administrator/.cuckoo/analyzer/windows/bin/cert.p12
mitmdump = /usr/local/bin/mitmdump 

Crtrl+C

enable in auxilliary.conf

It seem to have installed successfully without error, but now my question is how can I verify it is working correctly and can see encrypted traffic?  I've enabled it and did a simple https site URL submission with Cuckoo, then analyzed the pcap, but I'm not seeing anything in the SSL Stream.  Am I missing something?  Does anyone have any ideas on how I can verify it is actually working with cuckoo?  I would very much appreciate this.  Thanks!

**Comment by doomedraven** [comment]
you should get cert of mitmproxy and install in in vm(s) then just open in vm browser https://google.com

if you see it in mitmproxy, everything works just fine

**Comment by benrau87** [comment]
Just throwing something in but, mitmproxy doesn't work with rooter, correct? I thought you had to do the old 'static' routing.

**Comment by alias6466** [comment]
@doomedraven Just two questions doom.  Which cert store does the cert belong in?  Also does mitmproxy need to be running on my host machine, or can it just be installed, and the cert installed on the guest VM, and it will sniff the traffic when an analysis is initiated?  Just want to confirm.  Thanks.

**Comment by doomedraven** [comment]
install CA cert in trusted root, 
can be installed only, reading code is useful ;) 
https://github.com/cuckoosandbox/cuckoo/blob/master/cuckoo/auxiliary/mitm.py#L22

**Comment by alias6466** [comment]
@doomedraven actually i had the wrong interface defined for the forwarding iptables rules.  it's working now.  however, i noticed mitmproxy has to be open and running on the host for intercepting to work with cuckoo.  What is the preferred intercept mode for mitmproxy?  is it A for accept all or just nothing defined?  Thanks.

**Comment by doomedraven** [comment]
see link which i post, you have everything there and the required args
but that should start automatically if you enable it in aux conf

**Comment by alias6466** [comment]
ahh ok got ya, so all i need then is this script you posted copied into my local profile's mitm.py file?  correct?

**Comment by doomedraven** [comment]
No, just activate mitm in aux conf and done

**Comment by alias6466** [comment]
yep got that.  OK, so it seems to be working, but silently.  Is that intended?

**Comment by doomedraven** [comment]
?

**Comment by alias6466** [comment]
can you just confirm these the right steps, anything i'm missing?

sudo apt-get install python3-pip python3-dev libtiff5-dev libjpeg8-dev libwebp-dev
sudo pip3 install mitmproxy

mitmproxy
cd ~/.mitmproxy
cp mitmproxy-ca-cert.p12 /home/administrator/.cuckoo/analyzer/windows/bin/cert.p12
mitmdump = /usr/local/bin/mitmdump

Crtrl+C

install mitmproxy SSL cert on client

iptables -t nat -A PREROUTING -i vboxnet0 -p tcp --dport 80 -j REDIRECT --to-port 8080
iptables -t nat -A PREROUTING -i vboxnet0 -p tcp --dport 443 -j REDIRECT --to-port 8080

(vboxnet0 is my result server interface for cuckoo)

enable mitm in auxilliary.conf

like i said, it seems to be working but the mitmproxy terminal window is silent and not open.

**Comment by Nwinternights** [comment]
look at Portbase in auxiliary.conf . the default port in cuckoo for mitm is 50000 so your Iptables should be:
sudo iptables -t nat -A  PREROUTING -i vboxnet0 -p tcp -m tcp --dport 443 -j REDIRECT --to-ports 50000.
to be sure the port is listening when you start analysis:
netstat -anp | grep "portNumber".  
regards

**Comment by alias6466** [comment]
@Nwinternights Is there an easy way after getting this all set up to verify that Cuckoo can see HTTPS traffic?

**Comment by Nwinternights** [comment]
Download this sample:https://malwr.com/analysis/ZTBiODAzMGEwNGRkNGNmYzk4MjYyZThhMGNjOWYxMmI/
You should see a Post over https in network tab
![screenshot_20170828-230126](https://user-images.githubusercontent.com/18212777/29793793-95771d5c-8c45-11e7-87dd-dcaedc094c2c.png)

**Comment by alias6466** [comment]
@Nwinternights I don't get an HTTPS post, but I do get an HTTP post 400 status to pilateszenter.com.  Are you sure this malware sample has an HTTPS post?

**Comment by doomedraven** [comment]
is so easy as open `https://www.facebook.com` or `https://www.google.com` in url analysis, nothing else ;)

**Comment by alias6466** [comment]
@doomedraven I did that earlier, and just now - there was no HTTP(s) tab of network analysis screen, just HTTP... what could be wrong here?  

iptables rules are set as follows (flushed and reset them even)

sudo iptables -A FORWARD -o eth0 -i vboxnet0 -s 192.168.56.0/24 -m conntrack --ctstate NEW -j ACCEPT
sudo iptables -A FORWARD -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
sudo iptables -A POSTROUTING -t nat -j MASQUERADE

sudo iptables -t nat -A PREROUTING -i vboxnet0 -p tcp --dport 80 -j REDIRECT --to-port 50000
sudo iptables -t nat -A PREROUTING -i vboxnet0 -p tcp --dport 443 -j REDIRECT --to-port 50000

**Comment by doomedraven** [comment]
did you check urls in http tab?

**Comment by Nwinternights** [comment]
@alias6466  any news?

**Comment by alias6466** [comment]
I'm only getting HTTP posts when I submit an HTTPS URL...  What can I do next, since I've configured iptables with the rules stated above?

**Comment by doomedraven** [comment]
but you see the https url ? with params no?

**Comment by alias6466** [comment]
No, there's no HTTPS post under the network analysis HTTP(S) tab.

**Comment by alias6466** [comment]
They are all HTTP

**Comment by Nwinternights** [comment]
I didn't analyze code of the sample and u were right, it POsts in http and sometimes in https. 
this https://malwr.com/analysis/YjhiMjA0YmQyYTQzNGVjNmI0ZTllOWEwZTg5MzJmMDU/ does one GET in HTTPS.
by the way when the analysis is done can u post cuckoo and analyzer log and a screenshot of network tab? 
What version of mitm are u using ?  (pip show mitmproxy) The latest compatible for Python 2.7 is 0.18.2 
regards

**Comment by alias6466** [comment]
@Nwinternights 
Using mitmproxy 0.18.2 with python 2.7

Analyzer log
2017-09-01 09:39:12,140 [analyzer] DEBUG: Starting analyzer from: C:\tmpemad06
2017-09-01 09:39:12,140 [analyzer] DEBUG: Pipe server name: \??\PIPE\tbIesltxGhciNSkCT
2017-09-01 09:39:12,140 [analyzer] DEBUG: Log pipe server name: \??\PIPE\RlATUymSOQgMMqqzZHawyaO
2017-09-01 09:39:12,765 [analyzer] DEBUG: Started auxiliary module DbgView
2017-09-01 09:39:13,296 [analyzer] DEBUG: Started auxiliary module Disguise
2017-09-01 09:39:14,687 [analyzer] DEBUG: Started auxiliary module DumpTLSMasterSecrets
2017-09-01 09:39:14,733 [analyzer] DEBUG: Started auxiliary module Human
2017-09-01 09:39:15,437 [modules.auxiliary.installcert] INFO: Successfully installed PFX certificate.
2017-09-01 09:39:15,437 [analyzer] DEBUG: Started auxiliary module InstallCertificate
2017-09-01 09:39:15,437 [analyzer] DEBUG: Started auxiliary module Reboot
2017-09-01 09:39:15,467 [analyzer] DEBUG: Started auxiliary module RecentFiles
2017-09-01 09:39:15,515 [analyzer] DEBUG: Started auxiliary module Screenshots
2017-09-01 09:39:15,562 [analyzer] DEBUG: Started auxiliary module LoadZer0m0n
2017-09-01 09:39:17,875 [lib.api.process] INFO: Successfully executed process from path 'C:\\Program Files (x86)\\Microsoft Office\\Office16\\EXCEL.EXE' with arguments [u'C:\\Users\\nemesis\\AppData\\Local\\Temp\\df1105862aa8bf8f66c30a2109534c04fa13ff17336ea9101f433ce8c742a2b4.bin'] and pid 2440
2017-09-01 09:39:26,687 [modules.auxiliary.human] INFO: Found button u'&Yes', clicking it
2017-09-01 09:40:25,151 [modules.auxiliary.human] INFO: Closed Office window.
2017-09-01 09:40:27,542 [analyzer] INFO: Process with pid 2440 has terminated
2017-09-01 09:40:27,542 [analyzer] INFO: Process list is empty, terminating analysis.
2017-09-01 09:40:28,588 [analyzer] INFO: Analysis completed.



Cuckoo log
2017-09-01 09:39:49,483 [cuckoo.core.scheduler] INFO: Task #37: acquired machine COMPUTER8 (label=COMPUTER8)
2017-09-01 09:39:49,490 [cuckoo.auxiliary.mitm] INFO: Started mitm interception with PID 6463 (ip=192.168.56.1, port=50000).
2017-09-01 09:39:49,490 [cuckoo.core.plugins] DEBUG: Started auxiliary module: MITM
2017-09-01 09:39:49,496 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 6464 (interface=vboxnet0, host=192.168.56.101)
2017-09-01 09:39:49,497 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2017-09-01 09:39:49,519 [cuckoo.machinery.virtualbox] DEBUG: Starting vm COMPUTER8
2017-09-01 09:39:49,644 [cuckoo.machinery.virtualbox] DEBUG: Restoring virtual machine COMPUTER8 to Snapshot4
2017-09-01 09:39:52,792 [cuckoo.core.guest] INFO: Starting analysis on guest (id=COMPUTER8, ip=192.168.56.101)
2017-09-01 09:39:53,796 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:39:54,802 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:39:55,810 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:39:55,874 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:39:57,883 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:39:58,886 [cuckoo.core.guest] DEBUG: COMPUTER8: not ready yet
2017-09-01 09:40:10,926 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.7 (id=COMPUTER8, ip=192.168.56.101)
2017-09-01 09:40:34,968 [cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=COMPUTER8, ip=192.168.56.101, monitor=latest, size=3825109)
2017-09-01 09:41:23,275 [cuckoo.core.resultserver] DEBUG: LogHandler for live analysis.log initialized.
2017-09-01 09:41:27,978 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0001.jpg
2017-09-01 09:41:28,048 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 31794
2017-09-01 09:41:29,160 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0002.jpg
2017-09-01 09:41:29,171 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 44226
2017-09-01 09:41:32,288 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0003.jpg
2017-09-01 09:41:32,296 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 44589
2017-09-01 09:41:33,316 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0004.jpg
2017-09-01 09:41:33,323 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 44244
2017-09-01 09:41:35,169 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:41:35,410 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0005.jpg
2017-09-01 09:41:35,423 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 40762
2017-09-01 09:41:36,458 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0006.jpg
2017-09-01 09:41:36,472 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 41707
2017-09-01 09:41:38,581 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0007.jpg
2017-09-01 09:41:38,588 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 52919
2017-09-01 09:41:39,713 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0008.jpg
2017-09-01 09:41:39,721 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 63927
2017-09-01 09:41:41,177 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:41:41,180 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:41:47,187 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:41:47,190 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:41:53,198 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:41:53,200 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:41:59,207 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:41:59,210 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:05,214 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:05,217 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:11,223 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:11,225 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:17,233 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:17,235 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:22,253 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0009.jpg
2017-09-01 09:42:22,271 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 64317
2017-09-01 09:42:23,243 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:23,246 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:29,254 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:29,257 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:30,588 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0010.jpg
2017-09-01 09:42:30,668 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 64257
2017-09-01 09:42:35,264 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:35,267 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:36,269 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c772690>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:36,272 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:36,883 [cuckoo.core.resultserver] DEBUG: File upload request for shots/0011.jpg
2017-09-01 09:42:36,889 [cuckoo.core.resultserver] DEBUG: Uploaded file length: 44214
2017-09-01 09:42:42,279 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:42,282 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:43,285 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc26650>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:43,290 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:44,295 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d385d50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:44,299 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:45,306 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc12ad0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:45,310 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:46,315 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc26750>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:46,319 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:47,325 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b4ad79c90>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:47,328 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:53,337 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:42:53,340 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:54,345 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc802d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:54,350 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:55,353 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dcb3350>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:55,355 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:56,359 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dcb0410>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:56,366 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:57,371 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dcb3ad0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:57,377 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:58,381 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c844250>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:58,387 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:42:59,393 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c889f50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:42:59,398 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:05,407 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:43:05,414 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:06,418 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c7cc810>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:06,428 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:07,430 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ea150>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:07,432 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:08,437 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ff890>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:08,443 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:09,446 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c841610>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:09,448 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:10,453 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3eadd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:10,459 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:11,464 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c772b50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:11,466 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:17,472 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:43:17,479 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:18,483 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc80ed0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:18,490 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:19,495 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c772c50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:19,498 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:20,504 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c8446d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:20,510 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:21,515 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d38e410>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:21,519 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:22,524 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d38e190>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:22,530 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:23,533 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c772710>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:23,538 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:29,546 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:43:29,551 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:30,554 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dcb3290>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:30,558 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:31,562 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c879a10>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:31,568 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:32,571 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d385710>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:32,576 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:33,579 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3851d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:33,584 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:34,587 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ea190>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:34,591 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:35,594 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3dddd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:35,598 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:41,602 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:43:41,610 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:42,614 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d38e810>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:42,619 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:43,624 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c7fc410>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:43,629 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:44,635 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c8381d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:44,642 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:45,646 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c8384d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:45,656 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:46,663 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d38edd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:46,678 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:47,688 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c772dd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:47,690 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:53,697 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:43:53,700 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:54,703 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc12590>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:54,708 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:55,710 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c85bcd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:55,715 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:56,719 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc12410>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:56,724 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:57,728 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ff750>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:57,733 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:58,737 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c889dd0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:58,743 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:43:59,747 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc12a50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:43:59,753 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:05,766 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:44:05,772 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:06,778 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c844f50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:06,785 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:07,792 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dcb3350>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:07,798 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:08,802 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc6b1d0>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:08,809 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:09,814 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ff510>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:09,817 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:10,823 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3a1c10>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:10,829 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:11,834 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c89ff10>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:11,842 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:17,850 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:44:17,858 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:18,863 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c7cc290>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:18,866 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:19,870 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c841250>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:19,876 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:20,879 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c841a10>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:20,881 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:21,888 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c89ff90>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:21,896 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:22,900 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c85b950>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:22,905 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:23,910 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c841390>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:23,916 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:29,923 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),))
2017-09-01 09:44:29,927 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:30,932 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d3ffa50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:30,940 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:31,945 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3d385d50>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:31,950 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:32,956 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3c838810>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:32,962 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:33,967 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc26e90>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:33,974 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:34,978 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ConnectionError(MaxRetryError("HTTPConnectionPool(host='192.168.56.101', port=8000): Max retries exceeded with url: /status (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2b3dc26950>: Failed to establish a new connection: [Errno 111] Connection refused',))",),))
2017-09-01 09:44:34,982 [cuckoo.core.guest] DEBUG: COMPUTER8: analysis still processing
2017-09-01 09:44:35,984 [cuckoo.core.guest] INFO: COMPUTER8: end of analysis reached!
2017-09-01 09:44:36,001 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: MITM
2017-09-01 09:44:36,038 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Sniffer
2017-09-01 09:44:37,230 [cuckoo.machinery.virtualbox] INFO: Successfully generated memory dump for virtual machine with label COMPUTER8 to path /home/cuckoo/.cuckoo/storage/analyses/37/memory.dmp
2017-09-01 09:44:37,231 [cuckoo.machinery.virtualbox] DEBUG: Stopping vm COMPUTER8
2017-09-01 09:44:38,860 [cuckoo.core.scheduler] DEBUG: Released database task #37
2017-09-01 09:44:38,880 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #37
2017-09-01 09:44:38,880 [cuckoo.processing.behavior] WARNING: Analysis results folder does not contain any behavior log files.
2017-09-01 09:44:38,880 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #37
2017-09-01 09:44:38,881 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #37
2017-09-01 09:44:38,881 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #37
2017-09-01 09:44:38,881 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #37
2017-09-01 09:44:40,579 [cuckoo.processing.memory] DEBUG: Executing volatility 'pslist' module.
2017-09-01 09:44:40,914 [cuckoo.processing.memory] DEBUG: Executing volatility 'psxview' module.
2017-09-01 09:44:47,428 [cuckoo.processing.memory] DEBUG: Executing volatility 'callbacks' module.
2017-09-01 09:44:54,643 [cuckoo.processing.memory] DEBUG: Executing volatility 'ssdt' module.
2017-09-01 09:44:57,008 [cuckoo.processing.memory] DEBUG: Executing volatility 'timers' module.
2017-09-01 09:44:58,208 [cuckoo.processing.memory] DEBUG: Skipping 'messagehooks' volatility module
2017-09-01 09:44:58,208 [cuckoo.processing.memory] DEBUG: Executing volatility 'getsids' module.
2017-09-01 09:44:59,318 [cuckoo.processing.memory] DEBUG: Executing volatility 'privs' module.
2017-09-01 09:45:00,640 [cuckoo.processing.memory] DEBUG: Executing volatility 'malfind' module.
2017-09-01 09:45:05,596 [cuckoo.processing.memory] DEBUG: Skipping 'apihooks' volatility module
2017-09-01 09:45:05,596 [cuckoo.processing.memory] DEBUG: Executing volatility 'dlllist' module.
2017-09-01 09:45:07,561 [cuckoo.processing.memory] DEBUG: Executing volatility 'handles' module.
2017-09-01 09:45:32,241 [cuckoo.processing.memory] DEBUG: Executing volatility 'ldrmodules' module.
2017-09-01 09:45:42,611 [cuckoo.processing.memory] DEBUG: Executing volatility 'mutantscan' module.
2017-09-01 09:45:44,068 [cuckoo.processing.memory] DEBUG: Executing volatility 'devicetree' module.
2017-09-01 09:45:45,646 [cuckoo.processing.memory] DEBUG: Executing volatility 'svcscan' module.
2017-09-01 09:45:47,228 [cuckoo.processing.memory] DEBUG: Executing volatility 'modscan' module.
2017-09-01 09:45:48,298 [cuckoo.processing.memory] DEBUG: Executing volatility 'yarascan' module.
2017-09-01 09:45:48,298 [cuckoo.processing.memory] DEBUG: Executing volatility 'netscan' module.
2017-09-01 09:45:49,921 [cuckoo.core.plugins] DEBUG: Executed processing module "Memory" for task #37
2017-09-01 09:45:49,922 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #37
2017-09-01 09:45:49,922 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #37
2017-09-01 09:45:49,923 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #37
2017-09-01 09:45:50,012 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #37
2017-09-01 09:45:50,017 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #37
2017-09-01 09:45:50,021 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #37
2017-09-01 09:45:50,025 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #37
2017-09-01 09:45:50,149 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #37
2017-09-01 09:45:50,149 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #37
2017-09-01 09:45:50,152 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #37
2017-09-01 09:45:50,153 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=amsi_bypass minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,154 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=applocker_bypass minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,157 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=metasploit_shellcode minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,157 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powerfun minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,157 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_bitstransfer minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_c2dns minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_ddi_rc4 minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_dfsp minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_di minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_empire minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_meterpreter minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_reg_add minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powershell_unicorn minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,158 [cuckoo.core.plugins] DEBUG: You are running a version of Cuckoo that's not compatible with this signature (either it's too old or too new): cuckoo=2.0.4a5 signature=powerworm minversion=2.0.4 maxversion=None
2017-09-01 09:45:50,161 [cuckoo.core.plugins] DEBUG: Running 451 signatures
2017-09-01 09:45:50,305 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_handles_1
2017-09-01 09:45:50,305 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_ldrmodules_1
2017-09-01 09:45:50,306 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_malfind_2
2017-09-01 09:45:50,310 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_svcscan_1
2017-09-01 09:45:50,312 [cuckoo.core.plugins] DEBUG: Analysis matched signature: volatility_svcscan_3
2017-09-01 09:45:50,772 [cuckoo.core.plugins] DEBUG: Executed reporting module "JsonDump"

![network](https://user-images.githubusercontent.com/30936869/29976188-ce2ed3d6-8efe-11e7-99fb-43af9495ca2d.png)

**Comment by Nwinternights** [comment]
"2017-09-01 09:41:41,177 [cuckoo.core.guest] INFO: Virtual Machine /status failed (ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='192.168.56.101', port=8000): Read timed out. (read timeout=5)",),)) "  
Seems that something is wrong on agent.py. ensure that it starts on VM boot, gives to it admin rights, check Windows UAC - Firewall.
then boot the vm
and try from the cuckoo to telnet the VM IP on 8000, ping  VM <=> Cuckoo <=> vm
can you post the cuckoo.conf and virtualbox.conf?

**Comment by alias6466** [comment]
![uac](https://user-images.githubusercontent.com/30936869/30079230-211bb70a-9245-11e7-9720-bb34771fb998.png)
![pystart](https://user-images.githubusercontent.com/30936869/30079234-25cf7f34-9245-11e7-9132-f240fdc49929.png)


Confirmed my user is in Administrators group.  See attached conf files as well.

[cuckoo.conf]

[cuckoo]
# Enable or disable startup version check. When enabled, Cuckoo will connect
# to a remote location to verify whether the running version is the latest
# one available.
version_check = yes

# If turned on, Cuckoo will delete the original file after its analysis
# has been completed.
delete_original = no

# If turned on, Cuckoo will delete the copy of the original file in the
# local binaries repository after the analysis has finished. (On *nix this
# will also invalidate the file called "binary" in each analysis directory,
# as this is a symlink.)
delete_bin_copy = no

# Specify the name of the machinery module to use, this module will
# define the interaction between Cuckoo and your virtualization software
# of choice.
machinery = virtualbox

# Enable creation of memory dump of the analysis machine before shutting
# down. Even if turned off, this functionality can also be enabled at
# submission. Currently available for: VirtualBox and libvirt modules (KVM).
memory_dump = yes

# When the timeout of an analysis is hit, the VM is just killed by default.
# For some long-running setups it might be interesting to terminate the
# monitored processes before killing the VM so that connections are closed.
terminate_processes = no

# Enable automatically re-schedule of "broken" tasks each startup.
# Each task found in status "processing" is re-queued for analysis.
reschedule = no

# Enable processing of results within the main cuckoo process.
# This is the default behavior but can be switched off for setups that
# require high stability and process the results in a separate task.
process_results = yes

# Limit the amount of analysis jobs a Cuckoo process goes through.
# This can be used together with a watchdog to mitigate risk of memory leaks.
max_analysis_count = 0

# Limit the number of concurrently executing analysis machines.
# This may be useful on systems with limited resources.
# Set to 0 to disable any limits.
max_machines_count = 0

# Limit the amount of VMs that are allowed to start in parallel. Generally
# speaking starting the VMs is one of the more CPU intensive parts of the
# actual analysis. This option tries to avoid maxing out the CPU completely.
max_vmstartup_count = 10

# Minimum amount of free space (in MB) available before starting a new task.
# This tries to avoid failing an analysis because the reports can't be written
# due out-of-diskspace errors. Setting this value to 0 disables the check.
# (Note: this feature is currently not supported under Windows.)
freespace = 1024

# Temporary directory containing the files uploaded through Cuckoo interfaces
# (api.py and Django web interface). Defaults to the default temporary
# directory of the operating system (e.g., /tmp on Linux). Overwrite the value
# if you'd like to specify an alternative path.
tmppath = 

# Path to the unix socket for running root commands.
rooter = /tmp/cuckoo-rooter

[feedback]
# Cuckoo is capable of sending "developer feedback" to the developers so that
# they can more easily improve the project. This functionality also allows the
# user to quickly request new features, report bugs, and get in touch with
# support in general, etc.
enabled = no
name = 
company = 
email = 

[resultserver]
# The Result Server is used to receive in real time the behavioral logs
# produced by the analyzer.
# Specify the IP address of the host. The analysis machines should be able
# to contact the host through such address, so make sure it's valid.
# NOTE: if you set resultserver IP to 0.0.0.0 you have to set the option
# `resultserver_ip` for all your virtual machines in machinery configuration.
ip = 192.168.56.1

# Specify a port number to bind the result server on.
port = 2042

# Force the port chosen above, don't try another one (we can select another
# port dynamically if we can not bind this one, but that is not an option
# in some setups)
force_port = no

# Maximum size of uploaded files from VM (screenshots, dropped files, log).
# The value is expressed in bytes, by default 128 MB.
upload_max_size = 134217728

[processing]
# Set the maximum size of analyses generated files to process. This is used
# to avoid the processing of big files which may take a lot of processing
# time. The value is expressed in bytes, by default 128 MB.
analysis_size_limit = 134217728

# Enable or disable DNS lookups.
resolve_dns = yes

# Enable PCAP sorting, needed for the connection content view in the web interface.
sort_pcap = yes

[database]
# Specify the database connection string.
# NOTE: If you are using a custom database (different from sqlite), you have to
# use utf-8 encoding when issuing the SQL database creation statement.
# Examples, see documentation for more:
# sqlite:///foo.db
# postgresql://foo:bar@localhost:5432/mydatabase
# mysql://foo:bar@localhost/mydatabase
# If empty, defaults to a SQLite3 database at $CWD/cuckoo.db.
connection = 

# Database connection timeout in seconds.
# If empty, default is set to 60 seconds.
timeout = 60

[timeouts]
# Set the default analysis timeout expressed in seconds. This value will be
# used to define after how many seconds the analysis will terminate unless
# otherwise specified at submission.
default = 120

# Set the critical timeout expressed in (relative!) seconds. It will be added
# to the default timeout above and after this timeout is hit
# Cuckoo will consider the analysis failed and it will shutdown the machine
# no matter what. When this happens the analysis results will most likely
# be lost.
critical = 60

# Maximum time to wait for virtual machine status change. For example when
# shutting down a vm. Default is 60 seconds.
vm_state = 60











[virtualbox.conf]

[virtualbox]
# Specify which VirtualBox mode you want to run your machines on.
# Can be "gui" or "headless". Please refer to VirtualBox's official
# documentation to understand the differences.
mode = gui

# Path to the local installation of the VBoxManage utility.
path = /usr/bin/VBoxManage
# If you are running Cuckoo on Mac OS X you have to change the path as follows:
# path = /Applications/VirtualBox.app/Contents/MacOS/VBoxManage

# Default network interface.
interface = vboxnet0

# Specify a comma-separated list of available machines to be used. For each
# specified ID you have to define a dedicated section containing the details
# on the respective machine. (E.g. cuckoo1,cuckoo2,cuckoo3)
machines = COMPUTER8


[COMPUTER8]
# Specify the label name of the current machine as specified in your
# VirtualBox configuration.
label = COMPUTER8

# Specify the operating system platform used by current machine
# [windows/darwin/linux].
platform = windows

# Specify the IP address of the current virtual machine. Make sure that the
# IP address is valid and that the host machine is able to reach it. If not,
# the analysis will fail.
ip = 192.168.56.101

# (Optional) Specify the snapshot name to use. If you do not specify a snapshot
# name, the VirtualBox MachineManager will use the current snapshot.
# Example (Snapshot1 is the snapshot name):
snapshot = Snapshot4

# (Optional) Specify the name of the network interface that should be used
# when dumping network traffic from this machine with tcpdump. If specified,
# overrides the default interface specified in auxiliary.conf
# Example (vboxnet0 is the interface name):
interface = 

# (Optional) Specify the IP of the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the IP address for the Result Server as your machine sees it. If you don't specify an
# address here, the machine will use the default value from cuckoo.conf.
# NOTE: if you set this option you have to set result server IP to 0.0.0.0 in cuckoo.conf.
# Example:
resultserver_ip = 

# (Optional) Specify the port for the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the port for the Result Server as your machine sees it. If you don't specify a port
# here, the machine will use the default value from cuckoo.conf.
# Example:
resultserver_port = 

# (Optional) Set your own tags. These are comma separated and help to identify
# specific VMs. You can run samples on VMs with tag you require.
tags = 

# Mostly unused for now. Please don't fill it out.
options = 

# (Optional) Specify the OS profile to be used by volatility for this
# virtual machine. This will override the guest_profile variable in
# memory.conf which solves the problem of having multiple types of VMs
# and properly determining which profile to use.
osprofile = 


[honeyd]
# For more information on this VM please refer to the "services" section of
# the conf/auxiliary.conf configuration file. This machine is a bit special
# in the way that its used as an additional VM for an analysis.
# *NOTE* that if this functionality is used, the VM should be registered in
# the "machines" list in the beginning of this file.
label = honeyd
platform = linux
ip = 192.168.56.102
# The tags should at least contain "service" and the name of this service.
# This way the services auxiliary module knows how to find this particular VM.
tags = service, honeyd
# Not all services actually have a Cuckoo Agent running in the VM, for those
# services one can specify the "noagent" option so Cuckoo will just wait until
# the end of the analysis instead of trying to connect to the non-existing
# Cuckoo Agent. We can't really intercept any inter-VM communication from the
# host / gateway so in order to dump traffic between VMs we have to use a
# different network dumping approach. For this machine we use the "nictrace"
# functionality from VirtualBox (which is basically their internal tcpdump)
# and thus properly dumps inter-VM traffic.
options = nictrace noagent


Ping both ways from both cuckoo host and VM


telnet failed from cuckoo host to VM on 8000
$ telnet 192.168.56.101:8000
telnet: could not resolve 192.168.56.101:8000/telnet: Name or service not known

**Comment by alias6466** [comment]
@Nwinternights See my responses above

**Comment by Nwinternights** [comment]
2 things:
the correct syntax is  telnet 192.168.56.101 8000 
can you please remove on virtualbox.conf the snapshot4
like this
snapshot = Snapshot4 
snapshot = 
then start the guest VM do (ping, telnet, task manager if agent is running),  take new snapshot and shutdown the vm.
stop cuckoo instance and start again (cuckoo -d) and look on the logs if the new snapshot is loaded correctly EBUG: Restoring virtual machine COMPUTER8 to Snapshot5 or whatever u called the new snapshot).
submit the file.

**Comment by alias6466** [comment]
@Nwinternights OK, so I can telnet to my VM from host on port 8000.  But I cannot telnet to Host from VM on port 8000.  Both firewalls are disabled on Host and VM atm.

**Comment by Nwinternights** [comment]
It s normal because the agent is running only on  the guest machine on port 8000.  did you retake the snapshot after making all the checks?

**Comment by alias6466** [comment]
@Nwinternights Yes, just took another snapshot and then tested the malware sample.  Still only getting HTTP post.
